src/index.js

```diff
@@ -1,51 +1,135 @@
 
 import './style.css';
 const divContainer = document.querySelector(".books-container");
+const inputDiv = document.querySelector("#title");
+const inputSubmit = document.querySelector("#submit");
 
 
+let todoList = [];
+let isEdit = false
+let editId = null;
 
-const todoList = [
-    {
-      description: 'Go shopping for lunch',
-      completed: false,
-      index: 0,
-    },
-    {
-      description: 'Take kids to school',
-      completed: false,
-      index: 1,
-    },
-    {
-      description: 'Work on coding challenges',
-      completed: false,
-      index: 2,
-    },
-    {
-        description: 'Go to the gym',
-        completed: false,
-        index: 3,
-      },
-    {
-      description: 'Attend workshop',
-      completed: false,
-      index: 4,
-    },
-  ];
-
-  const displayTodo = ({ description, index }) => {
-    const divElement = document.createElement('div');
-    divElement.className = 'first-item';
-    divElement.innerHTML = `
+console.log(editId)
+
+
+
+  const setLocalStorage = (todoList)=>{
+    localStorage.setItem('formInputs', JSON.stringify(todoList));
+  }
+
+  const getLocalStorage = ()=>{
+    if (localStorage.getItem('formInputs') !== null) {
+      todoList = JSON.parse(localStorage.getItem('formInputs'));
+      } else {
+      todoList = [];
+
+      }
+      return todoList;
+  }
+
+  const displayTodo = ({ description, index}) => {
+    
+      const divElement = document.createElement('div');
+
+      divElement.className = 'first-item';
+      divElement.innerHTML = `
       <div class="item-details">
       <input type="checkbox" id="" name="" value=""> <h4 class="item-desription">${description}</h4>
       </div>
-      <i class="fa-solid fa-trash-can" id"${index}"></i>
-      `;
+      
+      <button class="edit">edit</button>
+      <button type="button" class="delete" id="${index}">del</button>
+            `;
   
     return divElement;
+    
+    
+  
   };
+
+  inputSubmit.addEventListener('click', (e) => {
+    e.preventDefault();
+      if(inputDiv.value.trim()!== ''){
+        
+          const list = {
+            description: inputDiv.value,
+            completed: false,
```

Comment:
Kindly fix your code so that when I check a checkbox, it toggles between true or false for the items in the local storage. Currently the items just remain on the default false. See the video attached.

https://user-images.githubusercontent.com/48546992/199699119-3da39869-f77a-458b-8b99-9353c576ae71.mov

Reply:
Fixed.